services/order.py

```diff
@@ -0,0 +1,33 @@
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, User, Ticket
+
+
+def create_order(
+        tickets: list,
+        username: str,
+        date: str = None
+) -> None:
+    with transaction.atomic():
+        new_order = Order.objects.create(
+            user=User.objects.get(username=username)
+        )
```

Comment:
due to checklist we should use get_user_model().objects "its the best practice" instead of User.objects...